docs/lib/auth/fragments/flutter/phone/add_mfa.md

```diff
@@ -0,0 +1,23 @@
+Turning MFA "ON" will make it required for all users, while "Optional" will make it available to enable on a per-user basis.
```

Comment:
*Question:* The majority of the docs below apply to iOS and Android as well (maybe even JS?), correct? Should these docs be added for all platforms, and then just the code snippets be added as fragments for each platform?

I am not sure if there is a norm for how this is done for totally new sections of docs

Reply:
For sure, yeah, and they can easily be turned on for other teams by just removing `platform="flutter"` but it didn't seem right to enable it for all teams at once.

Reply:
@Jordan-Nelson I've updated the tags to make it clear which ones are actually flutter-specific and moved the common text to a common folder.